### PR TITLE
Indent blocks to allow list number continuation

### PR DESCRIPTION
**Pull request by unode**
If I understood #125 correctly, this should fix it.

### CODE REVIEW

File: _episodes/starting-jekyll.md [hunk 1]
```diff
@@ -72,37 +72,40 @@ Let's make use of global parameters in our pages.
...
-{% raw %}{{ site.description }}{% endraw %}
+    ## Description
+    {% raw %}{{ site.description }}{% endraw %}
 
-Have any questions about what we do? [We'd love to hear from you!]({% raw %}mailto:{{ site.email }}{% endraw %})
-~~~  
-   
+    Have any questions about what we do? [We'd love to hear from you!]({% raw %}mailto:{{ site.email }}{% endraw %})
+    ~~~  
+
```
Comment (tobyhodges) [inline _episodes/starting-jekyll.md]:
```suggestion
    ~~~  
    {: .language-markdown }
    
```

File: _episodes/starting-jekyll.md [hunk 2]
```diff
@@ -72,37 +72,40 @@ Let's make use of global parameters in our pages.
...
 
-## Contact us
+    ## Contact us
 
-- Email: [{% raw %}{{ site.email }}{% endraw %}](mailto:{% raw %}{{ site.email }}{% endraw %})
-- Twitter: [@my_research_project](https://twitter.com/my_research_project)
-~~~
+    - Email: [{% raw %}{{ site.email }}{% endraw %}](mailto:{% raw %}{{ site.email }}{% endraw %})
+    - Twitter: [@my_research_project](https://twitter.com/my_research_project)
+    ~~~
```
Comment (tobyhodges) [inline _episodes/starting-jekyll.md:110]:
```suggestion
    ~~~
    {: .language-markdown }
```

File: _episodes/starting-jekyll.md [hunk 3]
```diff
@@ -115,35 +118,39 @@ You can cite the project as:
...
+> >     
+> >     You can cite the project as:
+> >     
+> >     > *My research project. Zenodo. https://zenodo.org/record/doi*
+> >     
+> >     ## Contact us
+> >     
+> >     - Email: [{% raw %}{{ site.email }}{% endraw %}](mailto:{% raw %}{{ site.email }}{% endraw %})
+> >     - Twitter: [{% raw %}{{ site.twitter }}{% endraw %}]({% raw %}{{ site.twitter }}{% endraw %})
+> >     ~~~
```
Comment (tobyhodges) [inline _episodes/starting-jekyll.md:155]:
```suggestion
> >     ~~~
> >     {: .language-markdown }
```

**Review by tobyhodges — APPROVED** [review]
Suggest to add Markdown syntax highlighting to these code blocks

**Comment by tobyhodges** [comment]
Closes #125

**Review by anenadic — APPROVED** [review]

**Pull request merged**